docs/upgrade-notes/upgrade-rds-cluster-tf-postgres-13.md

```diff
@@ -0,0 +1,43 @@
+---
+id: upgrade-rds-cluster-tf-postgres-13
+title: Upgrade Database Cluster to PostgreSQL v13
+hide_title: false
+---
+
+Cumulus Core as of version > 18.1.0** now supports and is tested against Aurora Postgres v13.   All users should update their datastores to this version as part of an upgrade process upon upgrading to release version 18.2.0.
+
+**This update was also backported to the 16.1.x release chain starting with v16.1.3.
+
+We recommend stopping all ingest rules if database downtime is required (e.g. you do not have a blue-green database solution or are using serverless V1) for the update  as any unavailability of the database may result in unexpected database write failures (resulting in records in the Dead Letter Archive), workflow failures or other unexpected failures.
+
+**It is also recommended that users test/evaluate the upgrade prior to performing it on their primary datastore in order to determine expected downtime and/or other related issues for their particular configuration.**
+
+## Users utilizing the `cumulus-rds-tf` module
+
+It is recommended that users manually backup and/or consider cloning their datastore in order to recover the datastore if an upgrade goes awry.
+
+Upgrading the Aurora Serverless v1 cluster will be completed via AWS console in this document and require manual steps to complete the upgrade:
+
+- Ensure a supported version (> 18.1.0 *or* 16.1.4 or a later patch version) is deployed.
+- Deploy the newest version of the `cumulus-rds-tf` module, ensuring `enable_upgrade` is set to false.   This will *only* deploy a `v13` version of your current parameter group configuration, named `<prefix>-cluster-parameter-group-v13`.
+- Shut down all ingest and other usage of the database cluster by 3rd party applications if appropriate.
```

Comment:
This should probably be moved up prior to the above "deploy" step.

Reply:
Is that right?   The module can be deployed with ingest up and an existing data store presuming no other changes.     I don't think the upgrade will start on deployment (the module can't support that), and at worst it would fail to deploy if the configuration params aren't set properly 🤔